var/spack/repos/builtin/packages/bumpversion/package.py

```diff
@@ -11,6 +11,7 @@ class Bumpversion(PythonPackage):
 
     pypi = "bumpversion/bumpversion-0.5.0.tar.gz"
 
+    version("0.6.0", sha256="4ba55e4080d373f80177b4dabef146c07ce73c7d1377aabf9d3c3ae1f94584a6")
     version("0.5.3", sha256="6744c873dd7aafc24453d8b6a1a0d6d109faf63cd0cd19cb78fd46e74932c77e")
     version("0.5.0", sha256="030832b9b46848e1c1ac6678dba8242a021e35e908b65565800c9650291117dc")
 
```

Comment:
Missing new bump2version dependency

Reply:
Since `bumpversion` is now just a shell that installs `bump2version` should we just replace the package?

Reply:
Up to you. If you want to get rid of the old package, just add `deprecated=True` to all versions.

Reply:
It looks like brew has kept the `bumpversion` naming and then just swapped the packages under the hood. Would that be an acceptable modification here too?

Reply:
Hmm, that seems like a weird choice. I would rather just have both packages to match pip.